### PR TITLE
Feature/other sda fuse tests

### PR DESCRIPTION
**Pull request by teemukataja**
- unit tests for `internal` package
- increased actions version from `1.17` to `1.17.7` because it was failing unit tests

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,8 +8,8 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0 // indirect
 	github.com/therecipe/qt v0.0.0-20200904063919-c0c124a5770d
-	github.com/therecipe/qt/internal/binding/files/docs/5.12.0 v0.0.0-20200904063919-c0c124a5770d // indirect
-	github.com/therecipe/qt/internal/binding/files/docs/5.13.0 v0.0.0-20200904063919-c0c124a5770d // indirect
 	golang.org/x/sys v0.0.0-20211102061401-a2f17f7b995c
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 )
+
+replace github.com/stretchr/testify => github.com/stretchr/testify v1.7.0
```
Comment (blankdots) [inline go.mod:15]: we might not need this any more as `github.com/stretchr/testify v1.7.0 // indirect` was added on line 9

Comment (blankdots) [reply go.mod:15]: actually we do, otherwise both v1.4.0 and v1.7.0 get install and the security notice will still be there, nevermind

Comment (emm1R) [reply go.mod:15]: The master branch does not have the replace command yet the security notice is no longer there.

**Review by blankdots — APPROVED** [review]
LGTM. there are some commented tests, not sure if those should be kept or will be tackled later.

**Comment by teemukataja** [comment]
> LGTM. there are some commented tests, not sure if those should be kept or will be tackled later.

@emm1R will add `filesystem` package tests here as well

**Comment by blankdots** [comment]
i merged #16 and it seems to have cause a dependency conflict, this will need a rebase

**Pull request merged**